Fix for formatting empty, malformed, or non json payload in error messages (#40889)
Azure/azure-sdk-for-python: sdk/cosmos/azure-cosmos/CHANGELOG.md

#### Breaking Changes

#### Bugs Fixed

> * Fixed Diagnostics Error Log Formatting from not accommodating error messages from none Cosmos Http Response Exceptions. See [PR 40889](https://github.com/Azure/azure-sdk-for-python/pull/40889/files)

Reviewer: ```suggestion
* Fixed Diagnostics Error Log Formatting to handle error messages from non-CosmosHttpResponseExceptions. See [PR 40889](https://github.com/Azure/azure-sdk-for-python/pull/40889/files)
```
